pkg/v6/resource/chartconfig/create.go

```diff
@@ -26,9 +26,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		}
 
 		for _, chartConfigToCreate := range chartConfigsToCreate {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chartconfig %s", chartConfigToCreate.Name))
+
 			_, err := guestG8sClient.CoreV1alpha1().ChartConfigs(resourceNamespace).Create(chartConfigToCreate)
 			if apierrors.IsAlreadyExists(err) {
-				// fall through
```

Comment:
Here would be good to have `did not create ...` log (described in the guide linked in other comment).